ZaeXT/test-scripts/test_failures.py

```
@@ -0,0 +1,183 @@
+    # 5. 【核心测试】Bob 尝试向 Alice 的对话中发送消息
+    print(f"  -> 尝试用 Bob 的身份向 Alice 的对话发送消息 (ID: {alice_conv_id})")
+    chat_payload = {"message": "这是一个恶意消息"}
+    res_bob_chat = requests.post(f"{BASE_URL}/conversations/{alice_conv_id}/messages", headers=bob_headers, json=chat_payload)
+    if res_bob_chat.status_code != 200:
+         print_success("权限隔离 - Bob 向 Alice 对话发消息被按预期拒绝")
+    else:
+         print_fail("权限隔离 - Bob 向 Alice 对话发消息的请求未被正确拒绝", res_bob_chat)
+
+    # 6. 验证 Alice 的对话仍然存在
+    print("  -> 验证 Alice 的对话仍然存在")
+    res_verify = requests.get(f"{BASE_URL}/conversations", headers=alice_headers)
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)